### ISSUE TITLE
Error while trying to generate a context diagram

### ISSUE THREAD

**Issue by rlebowitz** [ErikEJ/EFCorePowerTools#1592]
Describe what is not working as expected.

There appears to be an error thrown when trying to generate a dgml file for a DbContext for a Sqlite database.  This feature previously worked in vs2022 but when I try to run the EF Core Power Tool option for this with the latest version of the extension an error message appears in the status bar at the bottom of VS2022.

The message says to look in the Extensions Output window but there doesn't appear to be a View option for that window in the latest version of VS2022 so I can't supply the error message

- EF Core version EF Core 7

- Is Handlebars used: no

- Is T4 used: no

- Is .dacpac used: no

- EF Core Power Tools version: 2.5.1206.0

- Database engine: SQLite

- Visual Studio version: 2022 version 17.5.0 Preview 1.0

**Comment by ErikEJ** [comment]
@rlebowitz Look in the "Output" window for the Extensions pane:

![image](https://user-images.githubusercontent.com/4169187/203253962-d406df2f-7b90-4733-944a-ac3774e80cab.png)

In addition, look at the wiki for tips: https://github.com/ErikEJ/EFCorePowerTools/wiki/Inspect-your-DbContext-model#challenges

**Comment by rlebowitz** [comment]
Ah, that never occurred to me; I just assumed there was a separate pane.  I found the output:

System.InvalidOperationException: Error:
Error:
  An assembly specified in the application dependencies manifest (HebrewDb.Entities.deps.json) was not found:
    package: 'runtimepack.Microsoft.Windows.SDK.NET.Ref', version: '10.0.22621.27'
    path: 'Microsoft.Windows.SDK.NET.dll'

   at async Task EFCorePowerTools.Handlers.ModelAnalyzerHandler.GenerateAsync(string outputPath, Project project, GenerationType generationType)

What's odd here is that the file does appear to exist, here's a screen snippet from my solution explorer showing the output:
![Screenshot 2022-11-22 091912](https://user-images.githubusercontent.com/4519940/203351916-3071ba70-6dc2-4bfb-b4b7-a16d183c00f8.png)

**Comment by ErikEJ** [comment]
Run from the executable project, not the class library (as documented)

**Comment by rlebowitz** [comment]
Sorry, I had been experimenting with both approaches.  I have a separate console app that references the class library that I've also tried generating the diagram from.  Here's the Extension output:
System.InvalidOperationException: Error:
System.ArgumentException: No EF Core DbContext types found in the project
   at Modelling.EfCoreModelBuilder.GetDbContextTypes(DbContextOperations operations) in C:\Code\Github\EFCorePowerTools\src\GUI\efpt30.core\EFCoreModelBuilder.cs:line 87
   at Modelling.EfCoreModelBuilder.BuildResult(String outputPath, String startupOutputPath, Boolean generateDdl) in C:\Code\Github\EFCorePowerTools\src\GUI\efpt30.core\EFCoreModelBuilder.cs:line 37
   at Modelling.EfCoreModelBuilder.GenerateDebugView(String outputPath, String startupOutputPath) in C:\Code\Github\EFCorePowerTools\src\GUI\efpt30.core\EFCoreModelBuilder.cs:line 24
   at Modelling.Program.Main(String[] args) in C:\Code\Github\EFCorePowerTools\src\GUI\efpt30.core\Program.cs:line 60

I can see the HebrewDB.Entities.dll which contains the DbContext class.  I also made sure that I have a default constructor; public ExerciseContext()
        { }
Any other ideas?

**Comment by ErikEJ** [comment]
Nope. I need a repro (you can share privately via email)

**Comment by rlebowitz** [comment]
Well, this is definitely odd.

 

I started to make a separate Github repository for you to play with, copied the entity class library and the executable classes, etc. and lo and behold, I was able to generate the diagram.  There must be some slight difference in the projects in my actual solution that are affecting this.  I’ll see if I can solve this myself before bothering you again.  At least I have the means to generate a diagram which, as I suspected, revealed some mistakes in my database design.  The diagram feature is so useful…. Thanks for you work on this.
